akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -439,6 +515,12 @@ final class ClusterClientReceptionist(system: ExtendedActorSystem) extends Exten
     pubSubMediator ! DistributedPubSubMediator.Unsubscribe(topic, actor)
 
   /**
+   * Returns the underlying receptionist actor.
+   */
+  def underlying: ActorRef =
+    receptionist
```

Comment:
Ah yeah - nice one.

> On 9 May 2016, at 4:00 PM, Patrik Nordwall <notifications@github.com> wrote:
> 
> In akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala <https://github.com/akka/akka/pull/20462#discussion_r62452794>:
> 
> > @@ -439,6 +515,12 @@ final class ClusterClientReceptionist(system: ExtendedActorSystem) extends Exten
> >      pubSubMediator ! DistributedPubSubMediator.Unsubscribe(topic, actor)
> >  
> >    /**
> > +   * Returns the underlying receptionist actor.
> > +   */
> > +  def underlying: ActorRef =
> > +    receptionist
> make val receptionist public instead?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/akka/akka/pull/20462/files/f6056876db54608dc4b055f3af47295a87c594bf#r62452794>